src/SmartComponents/ResourceOptimization/ResourceOptimizationCard.js

```diff
@@ -0,0 +1,141 @@
+                                                <span className='pf-u-font-size-sm'>
+                                                Systems with suggestions
+                                                </span>
+                                            </Flex>
+                                            <Flex
+                                                direction={{ default: 'column' }}
+                                                spaceItems={{ default: 'spaceItemsNone' }}
+                                                alignItems={{ default: 'alignItemsCenter' }}>
+                                                {/* fixme: Need to update it to use values from backend once the API is updated  */}
+                                                <span className='pf-u-font-size-2xl pf-u-color-100 pf-u-font-weight-bold'>
+                                                15
+                                                </span>
+                                                <span className='pf-u-font-size-sm'>
+                                                Total systems
+                                                </span>
+                                            </Flex>
+                                        </Flex>
+                                        <Button
+                                            variant='secondary'
+                                            isSmall component='a'
+                                            onClick={e => navigateTo(e, `${UI_BASE}/ros`)}
+                                            href={ `${UI_BASE}/ros`}>
+                                            {intl.formatMessage(messages.rosCardViewSystemsCTA)}
+                                        </Button>
+                                    </Flex>
+
+                                </React.Fragment>
+                                : <Flex
+                                    direction={{ default: 'column' }}
+                                    alignItems={{ default: 'alignItemsCenter' }}
+                                >
+                                    <FlexItem>{ intl.formatMessage(messages.rosSystemsNotConfiguredMessage)}</FlexItem>
+                                    <Button
+                                        variant='secondary'
+                                        isSmall component='a'
+                                        onClick={e => navigateTo(e, `${UI_BASE}/ros`)}
+                                        href={ `${UI_BASE}/ros`}>
+                                        {intl.formatMessage(messages.rosCardConfigureSystemsCTA)}
+                                    </Button>
+                                </Flex>
+                        )
+
+                    }
+                    {rosIsConfiguredFetchStatus === 'pending' && <Loading />}
+                    {rosIsConfiguredFetchStatus === 'rejected' && <FailState appName='Resource optimization' />}
```

Comment:
I would translate the `Resource optimization` as well, but that's probably a product name, so let's not translate it.